unit_tests/UnitTestShmemAlignment.C

```diff
@@ -39,3 +39,8 @@ TEST(Shmem, align)
   EXPECT_EQ(3u, numCorrectlyAlignedViews);
 }
 
+TEST(Shmem, align)
```

Comment:
@alanw0 What is the reason for this extra level of indirection?

Reply:
@sayerhs By indirection do you mean pulling the body of the test out into a separate function? cuda complained that a device lambda can't be launched from within a class method that is private or protected. gtest TESTs are protected members of a class. So pulling the body of the test into a free-standing function allowed it to compile.